containers/azure-terraform/.vscode/tasks.json

```diff
@@ -0,0 +1,107 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "az login",
+            "type": "shell",
+            "command": "az login && az account set -s ${env:ARM_SUBSCRIPTION_ID}",
+            "problemMatcher": []
+        },
+        {
+            "label": "terraform create backend",
+            "type": "shell",
+            "linux": {
+                "command": "${workspaceFolder}/test-project/create-backend.sh"
+            },
+            "problemMatcher": []
+        },
+        {
+            "label": "terraform init",
+            "type": "shell",
+            "command": "terraform",
+            "problemMatcher": [],
+            "args": [
+                "init",
+                "-backend-config=\"storage_account_name=${env:TF_BACKEND_STORAGE_ACCOUNT}\"",
+                "-backend-config=\"container_name=${env:TF_BACKEND_CONTAINER}\"",
+                "-backend-config=\"key=${env:TF_BACKEND_KEY}\"",
+                "-backend-config=\"resource_group_name=${env:TF_BACKEND_RESOURCE_GROUP}\"",                              
+                "-backend-config=\"subscription_id=${env:ARM_SUBSCRIPTION_ID}\"",
```

Comment:
When user is authenticated via CLI `subscription_id` and `tenant_id` are not needed.

Reply:
Removed tenant_id and subscription_id from the `init` task